src/contact_form/contact_form.css

```
@@ -0,0 +1,106 @@
+.contact_form_section {
+    padding-top: 85px;
+    margin-bottom: -21px;
+}
+
+.contact_form_headers {
+    text-align: center;
+}
+
+.contact_form_header {
+    font-size: 36px;
+    font-weight: 600;
+    letter-spacing: .2px;
+}
+
+.contact_form_subheading {
+    padding-top: 3px;
+    color: #9F9F9F;
+    font-size: 16px;
+    font-weight: 400;
+    width: 644px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.contact_form_container {
+    padding: 140px 227px 0px 240px;
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+}
+
+.contact_item_container {
+    display: flex;
+    align-items: flex-start;
+    gap: 28px;
+    padding-bottom: 42px;
+}
+
+.contact_item_text_title {
+    font-size: 24px;
+    font-weight: 700;
+}
+
+.contact_item_text_text {
+    margin-top: -1px;
+    width: 212px;
+    font-size: 16px;
+    font-weight: 500;
+}
+
+.contact_form_item_inner {
+    display: flex;
+    flex-direction: column;
+}
+
+.form_item_label {
+    font-size: 16px;
+    font-weight: 500;
+}
+
+.form_item_input {
+    overflow: hidden;
+    margin-top: 21px;
+    margin-bottom: 36px;
+    width: 528.75px;
```

there is no much sense in using fractional px units
```suggestion
    width: 529px;
```